pkg/generate/app/cmd/newapp.go

```diff
@@ -856,3 +858,10 @@ func (c *AppConfig) HasArguments() bool {
 		len(c.Templates) > 0 ||
 		len(c.TemplateFiles) > 0
 }
+
+func (c *AppConfig) GetBuildEnvs(environment app.Environment) app.Environment {
```

Comment:
tiny minor nit... should be singular s/GetBuildEnvs/GetBuildEnv/ or even GetBuildEnvironment